types/react/package.json

```diff
@@ -2,5 +2,22 @@
   "private": true,
   "dependencies": {
     "csstype": "^3.0.2"
+  },
+  "exports": {
+    ".": {
+      "types": {
+        "default": "./index.d.ts"
```

Comment:
I've mostly just pre-formatted these as nested objects so the diff if/when an esm-native node entrypoint is added will be smaller.

Reply:
Cool